steps/step1_start_asterisk.go

```diff
@@ -0,0 +1,54 @@
+package steps
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+)
+
+func StartServAster() {
+	fmt.Println("Запуск сервера")
+	mux := http.NewServeMux()
+	mux.Handle("/", http.FileServer(http.Dir("./web")))
+	mux.HandleFunc("/api/nextdate", nextDateWeekMon)
+
+	mux.HandleFunc("/api/signin", auth)
+	mux.HandleFunc("/api/task", authTask(reqSelectAster))
+	mux.HandleFunc("/api/tasks/", authTask(searchHandler))

```

Comment:
Нужно явно указывать тип запроса, если мы это отдельно в функции не проверяем
```suggestion
	mux.HandleFunc("GET /api/tasks/", authTask(searchHandler))

```